cmd/dns.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+func dnsCommand() *cobra.Command {
+	versionCmd := &cobra.Command{
+		Use:   "dns",
+		Short: "DNS Operator command line utility",
+		Long:  "DNS Operator command line utility",
+		RunE:  runDNS,
+	}
+	return versionCmd
+}
+
+func runDNS(_ *cobra.Command, args []string) error {
+	// pass verbose from root
+	args = append(args, fmt.Sprintf("--verbose=%t", verbose))
+
+	out, err := exec.Command("kubectl-dns", args...).Output()
```

Comment:
does this mean that if I ran `kuadrantctl dns help` the output of `kubectl-dns help` would output?

Reply:
yep. [Here](https://github.com/Kuadrant/kuadrantctl/pull/117#discussion_r2288191522) is my output, but you are welcome to try it yourself